src/styles/helpers/__test__/clamp.spec.js

```diff
@@ -0,0 +1,24 @@
+import clamp from '../color/clamp';
+
+describe('clamp', () => {
+    it('should return the same value if it is between 0 and 1', () => {
+        expect.assertions(4);
```

Comment:
This is only needed when testing asyn code.
Take a look at this:
https://jestjs.io/docs/en/expect#expectassertionsnumber
https://jestjs.io/docs/en/asynchronous#promises